packages/ra-core/src/form/useInput.spec.tsx

```diff
@@ -208,30 +209,31 @@ describe('useInput', () => {
 
     it('does not apply the initialValue when input has a value of 0', () => {
         const { queryByDisplayValue } = renderWithRedux(
-            <FormWithRedirect
-                onSubmit={jest.fn()}
-                record={{ id: 1, views: 0 }}
-                render={() => {
-                    return (
-                        <Input
-                            source="views"
-                            resource="posts"
-                            initialValue={99}
-                        >
-                            {({ id, input }) => {
-                                return (
-                                    <input
-                                        type="number"
-                                        id={id}
-                                        aria-label="Views"
-                                        {...input}
-                                    />
-                                );
-                            }}
-                        </Input>
-                    );
-                }}
-            />
+            <RecordContextProvider value={{ id: 1, views: 0 }}>
```

Comment:
I don't understand why you need that change as you're adding a context in FormWithRedirect

Reply:
Forgot to fix this test after introducing the RecordContext in FormWithRedirect

Reply:
Not really important though